ert3/engine/_status.py

```diff
@@ -0,0 +1,22 @@
+import ert3
+
+
+def status(workspace_root):
```

Comment:
The knowledge that the status should be displayed in the terminal should be contained within the CLI. I suggest that we either move the entire function into `ert3.console` or introduce a view of the project that the engine can produce for which the CLI afterwards prints data from...